Krzysztof_Dziublewski_Latex_Homework.tex

```diff
@@ -0,0 +1,143 @@
+\documentclass{article}
+
+\title{My \LaTeX{} Homework}
+\author{Krzysztof Dziublewski}
+\date{21 May 2019}
+
+\usepackage{paracol}
+\usepackage{tipa}
+\usepackage{qtree}
+\usepackage{phonrule}
+\usepackage{polyglossia}
+\usepackage{churchslavonic}
+\setmainlanguage{english}
+\setotherlanguage{churchslavonic}
+\setotherlanguage{arabic}
+\newfontfamily{\cyrillicfont}{Monomakh Unicode TT}[Scale=MatchUppercase]
+\newfontfamily\arabicfont[Script=Arabic]{Noto Naskh Arabic}
+
+
+\begin{document}
+\maketitle
+My interests within Linguistics, NLP and CS:
+\begin{enumerate}
+    \item Text generation
+    \item Game development
+    \item Phonology
+    \item Cognitive linguistics
+    \item Sentiment analysis
+    \item Implicature extraction
+\end{enumerate}
+
+\textipa{"aI "nev@ "n@U w6t t@ seI wen "s2mb@di "A:sks mi @"baUt maI "IntrIsts s@U DIs "teIb@l Is "pr6b@bli nOt D@ m@Ust reprI"zent@tIv \textdoublepipe{} "s@U It "g@Uz \textdoublepipe{} \textsubbridge{s}\textbari{}\textsubbridge{t}u"wA\texttoptiebar{t\textsubbridge{s}}\super jA "mA "CE pO"\textsubbridge{d}Ob\textltailn{}E "g\textsubbridge{d}\textbari{} "mAm "\texttoptiebar{t\textsubbridge{s}}OC "O "sOb\super jjE pO"v\super jjE\texttoptiebar{d\textctz{}}E\texttoptiebar{tC} \textdoublepipe{} "\textsubbridge{z}\super jjA\textsubbridge{d}wEm "\texttoptiebar{d\textctz{}}iCAj "bAr\texttoptiebar{d\textsubbridge{Z}O} "\textsubbridge{s}mA\texttoptiebar{tS}n\textbari{} "Ob\super jjA\textsubbridge{d} "i "x\texttoptiebar{t\textsubbridge{s}}E "mi "CE "\textsubbridge{s}pA\texttoptiebar{tC}\textdoublepipe{} "\textsubbridge{t}\textbari{}lE "mAm f "sum\super jjE \textsubbridge{d}o pOv\super jjE"\texttoptiebar{d\textsubbridge{z}}E\textltailn{}a \textdoublepipe{}}
```

Comment:
Very small details:
"O > O
"i > i (prepositions are not stressed)
Polish /a/ is /a/ (central), not /A/ (back)
obiad - final devoicing